clusterloader2/pkg/flags/flags.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flags
+
+import (
+	"flag"
+	"os"
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+)
+
+const (
+	noInt    = -12345
+	noString = "xxxNoStringxxx"
```

Comment:
I'm not a fan of those strange defaults. Defaults are part of documentation (in fact they are kind of part of api also). So I don't want to have stuff like that.
I would really prefer using standard defaults (empty values) - which always has to be overwritten anyway.

Reply:
I changed consts to 0 and "" respectively.
I though about having some magic variables that user will definitely not use. It was useful to verifying whether flag was provided or not.